docs/integrators/snapshotless-observing-squad.md

```diff
@@ -0,0 +1,68 @@
+---
+id: snapshotless-observing-squad
+title: Snapshotless Observing Squad
+---
+
+[comment]: # (mx-abstract)
+
+This page describes the Snapshotless Observing Squad, a type of Observing Squad optimized for real-time requests such as accounts data fetching and vm-query operations.
+
+[comment]: # (mx-context-auto)
+
+## Overview
+
+Whenever a node is executing the trie snapshotting process, the accounts data fetching & vm-query operations are greatly affected.
```

Comment:
`can be greatly affected` / `the performance of ... can be negatively impacted` (less strong phrasing).

Reply:
done